demo/src/screens/SettingsScreen.js

```diff
@@ -99,10 +117,13 @@ class SettingsScreen extends Component {
               <Text text80 dark20>Force RTL</Text>
             </View>
           </View>
+          
+          {extraSettingsUI?.()}
```

Comment:
Why do you render extraSettingsUI twice?

Reply:
fixed.
(public/private copy mistake)

Reply:
👍